lighthouse-cli/chrome-launcher.ts

```diff
@@ -70,10 +70,16 @@ class ChromeLauncher {
   flags() {
     const flags = [
       `--remote-debugging-port=${this.port}`,
-      '--disable-extensions',
       '--disable-translate',
-      '--disable-default-apps',
-      '--no-first-run',
+      '--disable-extensions', // disable all chrome extensions entirely
+      // '--ignore-certificate-errors', // Ignore certificate errors, like self-signed
```

Comment:
removed ignore-certificate-errors from this PR. 
filed #1443 for --chrome-flags